libs/datamodel/core/src/lib.rs

```diff
@@ -85,8 +85,8 @@ pub mod json;
 pub mod transform;
 pub mod walkers;
 
+pub use crate::dml::*;
 pub use configuration::*;
-pub use dml::*;
```

Comment:
This should still work with `pub use dml::*` imo. Would be better to stick with one style.

Reply:
`pub use dml::*` does not work in this case, since there is both the extern crate dml and the module dml, so this would be ambiguous

Reply:
Ah right!